pyaedt/filtersolutions_core/lumped_termination_impedance.py

```diff
@@ -0,0 +1,391 @@
+        self.table_type = table_type
+
+    def _define_termination_impedance_dll_functions(self):
+        """Define C++ API DLL functions."""
+        self._dll.getComplexTableRowCount.argtypes = [POINTER(c_int), c_bool]
+        self._dll.getComplexTableRowCount.restype = c_int
+
+        self._dll.getComplexTableRow.argtypes = [
+            c_int,
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+            c_int,
+        ]
+        self._dll.getComplexTableRow.restype = c_int
+
+        self._dll.updateComplexTableRow.argtypes = [
+            c_int,
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.updateComplexTableRow.restype = c_int
+
+        self._dll.appendComplexTableRow.argtypes = [
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.appendComplexTableRow.restype = c_int
+
+        self._dll.insertComplexTableRow.argtypes = [
+            c_int,
+            c_char_p,
+            c_char_p,
+            c_char_p,
+            c_bool,
+        ]
+        self._dll.insertComplexTableRow.restype = c_int
+
+        self._dll.removeComplexTableRow.argtypes = [c_int, c_bool]
+        self._dll.removeComplexTableRow.restype = c_int
+
+        self._dll.setLumpedComplexDefinition.argtypes = [c_char_p, c_bool]
+        self._dll.setLumpedComplexDefinition.restype = c_int
+        self._dll.getLumpedComplexDefinition.argtypes = [c_char_p, c_bool, c_int]
+        self._dll.getLumpedComplexDefinition.restype = c_int
+
+        self._dll.setLumpedComplexReactanceType.argtypes = [c_char_p, c_bool]
+        self._dll.setLumpedComplexReactanceType.restype = c_int
+        self._dll.getLumpedComplexReactanceType.argtypes = [c_char_p, c_bool, c_int]
+        self._dll.getLumpedComplexReactanceType.restype = c_int
+
+        self._dll.setLumpedComplexImpCompensateEnabled.argtypes = [c_bool, c_bool]
+        self._dll.setLumpedComplexImpCompensateEnabled.restype = c_int
+        self._dll.getLumpedComplexImpCompensateEnabled.argtypes = [
+            POINTER(c_bool),
+            c_bool,
+        ]
+        self._dll.getLumpedComplexImpCompensateEnabled.restype = c_int
+
+        self._dll.setLumpedComplexCompOrder.argtypes = [c_int, c_bool]
+        self._dll.setLumpedComplexCompOrder.restype = c_int
+        self._dll.getLumpedComplexCompOrder.argtypes = [POINTER(c_int), c_bool]
+        self._dll.getLumpedComplexCompOrder.restype = c_int
+
+    def table_type_to_bool(self):
+        """Set a flag to recognize source and load complex tables.
+
+        Returns
+        -------
+        bool
+        """
+        if self.table_type.value == TerminationType.SOURCE.value:
+            return False
+        else:
+            return True
+
+    @property
+    def row_count(self) -> int:
+        """The count of accumulated complex impedances in the complex impedances tables.
```

Comment:
```suggestion
        """Count of the accumulated complex impedances in the complex impedance's tables.
```
Possessive here assumes a singular table. If more than one table, use "impendances' tables" instead. It might be sufficient to not use possessive at all but rather use "in the complex impendance tables."